ray-operator/apis/ray/v1alpha1/events/raycluster_events.go

```diff
@@ -0,0 +1,5 @@
+package v1alpha1
+
+const (
+	Unhealthy = "RayUnhealthy"
```

Comment:
Following [K8s events](https://github.com/kubernetes/kubernetes/blob/89aaf7c02dd33415ac994cecfddca7c02ed75834/pkg/kubelet/events/event.go) convention of being one word, camel-cased.

Reply:
We might eventually want to indicate different kinds of unhealthy.
Would something more specific like `RayConfigError` be better?

Reply:
sgtm, updated